### PR TITLE
Improve Khoj Chat and Settings UI

### PR DESCRIPTION
**Pull request by debanjum**
- 707c6950 Disable Whatsapp config section on web client if Twilio not configured
- 7e23ffd6 Render line breaks in chat messages to improve formatting on web, desktop apps
- 7241459a Use single debug mode check function. Disable telemetry in debug mode
- 49076f0d Fix license in pyproject.toml. Use logger.error instead of print

### CODE REVIEW

File: src/khoj/processor/embeddings.py [hunk 1]
```diff
@@ -56,8 +56,8 @@ def embed_documents(self, docs):
                     try:
                         response.raise_for_status()
                     except requests.exceptions.HTTPError as e:
-                        print(f"Error: {e}")
-                        print(f"Response: {response.json()}")
+                        logger.error(f"Error: {e}")
```
Comment (sabaimran) [inline src/khoj/processor/embeddings.py:59]: Can remove this change after I merged #628 .

File: src/khoj/routers/api_config.py [hunk 2]
```diff
@@ -294,6 +294,7 @@ async def update_phone_number(
         create_otp(user)
     else:
         logger.warning("Phone verification is not enabled")
+        return {"status": "unsupported"}
```
Comment (sabaimran) [inline src/khoj/routers/api_config.py:297]: No point in setting the number at all if Twilio isn't enabled, right?

**Review by sabaimran — COMMENTED** [review]
Some minor comments around clean-up, but looks goooooood.

File: src/khoj/processor/embeddings.py (hunk 1 above)
Comment (debanjum) [reply src/khoj/processor/embeddings.py]: Removed that

File: src/khoj/interface/web/config.html [hunk 3]
```diff
@@ -610,15 +612,21 @@ <h3 class="card-title">
...
-    let isPhoneNumberVerified = "{{ is_phone_number_verified }}";
-    let isTwilioEnabled = "{{ is_twilio_enabled }}";
+    const preExistingPhoneNumber = "{{ phone_number }}";
+    const isPhoneNumberVerified = "{{ is_phone_number_verified }}";
+    const isTwilioEnabled = "{{ is_twilio_enabled }}";
+    isPhoneNumberVerified = isPhoneNumberVerified.toLowerCase();
+    isTwilioEnabled = isTwilioEnabled.toLowerCase();
 
-    if (preExistingPhoneNumber != "None" && (isPhoneNumberVerified == "True" || isPhoneNumberVerified == "true")) {
+    if (isTwilioEnabled !== "true" ) {
```
[the remark this answers is not in the document]
Comment (debanjum) [reply src/khoj/interface/web/config.html:621]: Fair point. Updated code to only enable to /config/phone API if Twilio is setup on server

**Review by sabaimran — APPROVED** [review]

**Pull request merged**